README.md

```diff
@@ -17,21 +17,66 @@ pipx install tap-googleads
 
 ## Configuration
 
-### Get refresh token
-1. GET https://accounts.google.com/o/oauth2/v2/auth?response_type=code&client_id=client_id&redirect_uri=http://127.0.0.1&scope=https://www.googleapis.com/auth/adwords&state=autoidm&access_type=offline&prompt=select_account&include_granted_scopes=true
-1. POST https://www.googleapis.com/oauth2/v4/token?code={code}&client_id={client_id}&client_secret={client_secret}&redirect_uri=http://127.0.0.1&grant_type=authorization_code
-1. POST https://www.googleapis.com/oauth2/v4/token?refresh_token={refres_token}&client_id={client_id}&client_secret={client_secret]&grant_type=refresh_token
 ### Accepted Config Options
 
 - [ ] `Developer TODO:` Provide a list of config options accepted by the tap.
 
-A full list of supported settings and capabilities for this
-tap is available by running:
+### This tap supports two sets of configs:
+
+### Using Your Own Credentials
+
+Settings required to run this tap.
+
+- `client_id` (required)
```

Comment:
We should just use the `--about --format=markdown` here and paste the table into the ReadMe. Much easier for everyone!

Reply:
I completely forgot about this, much better idea. The only minor issue is the lack of required settings specified in the tap as we have 2 ways to provide auth now. (See my other comment for more info).